conference/management/commands/createsuperuser.py

```diff
@@ -0,0 +1,176 @@
+"""
```

Comment:
not quite sure about overwriting this one. any pros and cons vs just adding another (and simplified) `create_ep_user.py` or something like that? I'd like to avoid copying code from django, just to customise it a little. ;)

Reply:
I agree, I'm not a fan of duplicating code - I was looking for a way of avoiding doing that, but came up empty handed. What I looked at was:
1. A way of overriding `User._default_manager` so that it's picked up by the default `createsuperuser` command. Since we don't use a custom `User` class, there's no way to override the manager though.
1. A way of extending from the Django `createsuperuser` command to create an `assopy.User` associated with the admin user created. The command does not return the user though; I could fetch the last created superuser, but that introduces a race condition.

That's why a new command was needed in the first place. I went with overriding the Django one as any new developer when starting on a Django application would use the `createsuperuser` to try to create an admin user; for existing developer it could also be easy to forget there's a special command that has to be used for this purpose; I think having completely separate command could lead to confusion. The downside is of course code duplication.

Now that I think about it some more, I guess maybe the option 1. from the list above would be doable, although not pretty - It would require creating a custom Django user class from `AbstractBaseUser`, having it use the existing `auth.User` db table; this way its `Manager` methods can be overriden. Thoughts?